### PR TITLE
Fixed em-dash split issue for bing_linkedin_contacts.py

### PR DESCRIPTION
**Pull request by chriswessells**
**Before submitting a pull request, make sure to complete the following:**
- [x] Ensure there are no similar pull requests.
- [x] Read the [Development Guide](https://github.com/lanmaster53/recon-ng/wiki/Development-Guide).

**What kind of PR is this?**  
_Please add an 'x' in the appropriate box, and apply a label to the PR matching the type here._
- [x] Bug Fix
- [ ] New Module
- [ ] Documentation Update

**Checklist For Approval**
- [x] Updated the meta dictionary for the module. NA
  - [x]If bug fix, updated the version.
- [x] Indexed the module NA
- [x] Added the index to the `modules.yml` file NA
- [x] Made the most out of the available [mixins](https://github.com/lanmaster53/recon-ng/wiki/Development-Guide#mixins). NA
- [x] Ensured the code is PEP8 compliant with `pycodestyle` or `black`.

### CODE REVIEW

**Comment by chriswessells** [comment]
Closes #198

File: modules/recon/profiles-contacts/bing_linkedin_contacts.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+import re
 
 
 class Module(BaseModule, BingAPIMixin):
 
     meta = {
         "name": "Bing LinkedIn Profile Contact Harvester",
         "author": "Cam Barts (@cam-barts)",
-        "version": "1.1",
+        "version": "1.2",
```
Comment (sig1nt) [inline modules/recon/profiles-contacts/bing_linkedin_contacts.py:12]: Looks like you bumped the version, but didn't run the `index.py` script to change the `modules.yml`. Would you mind doing that?

File: modules/recon/profiles-contacts/bing_linkedin_contacts.py [hunk 2]
```diff
@@ -35,7 +36,9 @@ def get_contact_info(self, url):
             # Split the title on the pipe to get rid of "linkedIn" portion at the end
             name_and_title = link_title.split("|")[0]
             # Split whats left on the Dashes, which is usually name - title - company
-            name_title_company_list = name_and_title.split("-")
+            # some european LinkedIn sites use em-dash
+            delimeter_expression = '- | ' + b'\xe2\x80\x93'.decode('utf-8')
```
Comment (sig1nt) [inline modules/recon/profiles-contacts/bing_linkedin_contacts.py:40]: Can we have the em-dash code as a constant for readability?

File: modules/recon/profiles-contacts/bing_linkedin_contacts.py (hunk 1 above)
Comment (chriswessells) [reply modules/recon/profiles-contacts/bing_linkedin_contacts.py:12]: You bet

File: modules/recon/profiles-contacts/bing_linkedin_contacts.py (hunk 2 above)
Comment (chriswessells) [reply modules/recon/profiles-contacts/bing_linkedin_contacts.py:40]: Yes, that makes sense.

**Comment by chriswessells** [comment]
I made the requested changes in the latest commit.

File: modules/recon/profiles-contacts/bing_linkedin_contacts.py [hunk 3]
```diff
@@ -35,7 +36,9 @@ def get_contact_info(self, url):
             # Split the title on the pipe to get rid of "linkedIn" portion at the end
             name_and_title = link_title.split("|")[0]
             # Split whats left on the Dashes, which is usually name - title - company
-            name_title_company_list = name_and_title.split("-")
+            # some european LinkedIn sites use em-dash
+            DELIMETER_EXPRESSION = '- | ' + b'\xe2\x80\x93'.decode('utf-8')
```
Comment (sig1nt) [inline modules/recon/profiles-contacts/bing_linkedin_contacts.py:40]: Apologies for a lack of clarity, I meant specifically moving the `b'\xe2\x80\x93'.decode('utf-8')` into a constant named something like `EM_DASH`. Though if you want to name the re expression, be my guest.

**Comment by sig1nt** [comment]
Awesome, thanks :)

**Review by lanmaster53 — APPROVED** [review]

**Pull request merged**